src/CoreGraphics/CGPattern.cs

```diff
@@ -53,12 +53,21 @@ public enum CGPatternTiling {
 	delegate void DrawPatternCallback (/* void* */ IntPtr info, /* CGContextRef */ IntPtr c);
 	delegate void ReleaseInfoCallback (/* void* */ IntPtr info);
 
+#if NET
+	[StructLayout (LayoutKind.Sequential)]
+	unsafe struct CGPatternCallbacks {
+		internal /* unsigned int */ uint version;
+		internal delegate* unmanaged<IntPtr, IntPtr, void> draw;
+		internal delegate* unmanaged<IntPtr, void> release;
+	}
+#else
 	[StructLayout (LayoutKind.Sequential)]
 	struct CGPatternCallbacks {
 		internal /* unsigned int */ uint version;
 		internal DrawPatternCallback draw;
 		internal ReleaseInfoCallback release;
 	}
+#endif
```

Comment:
Same comment about the struct. 
```suggestion
	[StructLayout (LayoutKind.Sequential)]
	struct CGPatternCallbacks {
		internal /* unsigned int */ uint version;
#if NET
		internal delegate* unmanaged<IntPtr, IntPtr, void> draw;
		internal delegate* unmanaged<IntPtr, void> release;
#else 
		internal DrawPatternCallback draw;
		internal ReleaseInfoCallback release;
	}
#endif
```